fpp

```diff
@@ -25,6 +25,9 @@ PYTHONCMD="python"
 function doProgram {
   # process input from pipe and store as pickled file
   $PYTHONCMD "$BASEDIR/src/processInput.py" "$@"
+  # if it failed, just fail now and exit the script
+  # this works for the looping -ko case as well
+  if [[ $? != 0 ]]; then exit $?; fi
```

Comment:
ah nice -- i never figured out how to exit with the same status code as the previous command!

I bet we could probably remove the code that clears out the bash script as well after this change...